Ensure all co-authors are displayed

andersonvom opened the pull request:
The last author was being ommited whenever the file did not have an
empty line at the end. This ensures every author is included, regardless
of trailing empty line.

ryanjwessel commented on .pear/pear.sh:
```diff
@@ -16,13 +16,7 @@ if [ "$SHA1" ]; then
-    n=$((n+1))
-done < $filename
-coauthors="$(echo $coauthors | sed 's/,$//g')" # Remove trailing comma
+coauthors="Co-authors: $(paste -sd "," $filename | sed -e 's/,$//')"
```
Nice! This looks like it works to me.

One thing I noticed is the previous output was: 

```
Co-authors: Ryan Wessel, Master Chief, Gandalf the Grey, Tony Stark
```

And with this change, it looks like this:

```
Co-authors: Ryan Wessel,Master Chief,Gandalf the Grey,Tony Stark
```

[In the matrix command](https://github.com/ryanjwessel/pear-cli/blob/main/src/commands/matrix.ts#L76), we split these strings by `, ` (note the space after the comma), so this would prevent the pairs from being split correctly.

ryanjwessel replied:
I'm not sure if there is a straightforward way to include the trailing space in this one-liner, or we could update the matrix command to no longer split by `, `.  It seems like updating the matrix command would be preferable?

It could be something like:

```js
body.split(CO_AUTHORS)[1].split(',').map(pair => pair.trim())
```

I think replacing new lines in that command was a leftover from a previous attempt at doing this, and all we actually have to do is split by commas then trim any excess whitespace.

ryanjwessel replied:
Also, I realize it is a bit unclear, but this is the generated shell script file. The file that is used for publishing is found in `src/pear.sh`.

Commit:
Fix missing last author issue

Previously, if the session file didn't have an empty last line, the last
author wouldn't be picked up in the while loop and would be omitted from
the `Co-authors` string.

`paste` concatenates all lines in a single one, ensuring that we don't
miss any authors.

Co-authors: Ryan Wessel

Commit:
Extract coAuthors function

This searches for a `Co-authors` line in the commit message and returns
a list of authors in case it exists, or an empty list otherwise.

Co-authors: Ryan Wessel

Commit:
Add contributor

Co-authors: Ryan Wessel

Commit:
Update local pear setup

Equivalent to running `yarn pear init`.

Co-authors: Ryan Wessel

Commit:
Bump version from 2.4.11 to 2.4.12
